src/jackknife.jl

```diff
@@ -0,0 +1,155 @@
+"""
+    Delete 1 jackknife with  for unstratified designs.
+    
+    Replicate weights are given by:
+    wi(hj)  = 0, if observation unit i is in psu j of stratum h
+            = nh(nh −1)wi, if observation unit i is in stratum h but not in psu j.
+    ## Reference
+    pg 380-382, Section 9.3.2 Jackknife - Sharon Lohr, Sampling Design and Analysis (2010)
+"""
+function jk1weights(design::SurveyDesign)
+    cluster_sorted = sort(design.data, design.cluster) # Does jk need sorting?
+    psus = unique(cluster_sorted[!, design.cluster])
+    nh = length(psus)
+    npsus = [(count(==(i), cluster_sorted[!, design.cluster])) for i in psus]       
+    replicates = [filter(n -> n != i, 1:nh) for i in 1:nh]
+    cluster_weights = cluster_sorted[!, design.weights]
+    for (i,replicate) in enumerate(replicates)
+        cluster_sorted[!, "replicate_"*string(i)] = vcat(
+            [   fill((count(==(i), replicate)) .* (nh / (nh - 1)), npsus[i]) for i = 1:nh
+            ]...,
+        ) .* cluster_weights
+    end
+    return ReplicateDesign(
+        cluster_sorted,
+        design.cluster,
+        design.popsize,
+        design.sampsize,
+        design.strata,
+        design.weights,
+        design.allprobs,
+        design.pps,
+        length(replicates),
+    )
+end
+
+"""
+    Jackknife estimator
+
+    Uses jackknife replicate weights to calculate variance of estimator θ̂
+"""
+function JackknifeEstimator(θ̂::Float64,design::ReplicateDesign)
+    gdf = groupby(design.data,design.strata)
+    # @show keys(gdf)
+    calc_df = combine(x -> length(unique(x[!,design.cluster])), gdf)
+    calc_df[!,:multiplier] = (calc_df[!,:x1] .- 1 ) ./ calc_df[!,:x1]
+    # for each_gdf in gdf
+    #     @show nrow(each_gdf)
+    # end
+    return calc_df
+end
```

Comment:
Don't need this. It's common for all replicate methods.

Reply:
i dont think its the same. See #260

Reply:
See Lohr, pg 382, formula 9.9 on how the Jackknife estimator uses the replicate weights columns differently to bootstrap